R/geom-rug.r

```diff
@@ -76,7 +80,10 @@ geom_rug <- function(mapping = NULL, data = NULL,
 GeomRug <- ggproto("GeomRug", Geom,
   optional_aes = c("x", "y"),
 
-  draw_panel = function(data, panel_params, coord, sides = "bl", outside) {
+  draw_panel = function(data, panel_params, coord, sides = "bl", length, outside) {
+    if (!is(length, "unit")) {
+      stop("'length' must be a 'unit' object.")
```

Comment:
Please add `call. = FALSE` to suppress verbose tracebacks. c.f https://style.tidyverse.org/error-messages.html